modules/effects/src/main/scala/dev/profunktor/redis4cats/effects.scala

```diff
@@ -208,4 +207,29 @@ object effects {
     /** Set expiry only when the new expiry is greater than current one */
     case object Lt extends ExpireExistenceArg
   }
+
+  sealed abstract class RedisType(val asString: String)
```

Comment:
I would add a link to the **Redis** docs: https://redis.io/docs/latest/develop/data-types/